### PR TITLE
Likes

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check

`src/components/likedcard/LikedCard.jsx`
`src/context/like-context.js`
`src/page/liked/Liked.jsx`
`src/reducer/liked-reducer.js`

## [Live](https://deploy-preview-5--play-arts.netlify.app/)

### CODE REVIEW

File: src/components/aside/aside.module.css [hunk 1]
```diff
@@ -1,14 +1,18 @@
 .asideUl {
-  text-align: start;
-  font-size: large;
+  font-size: 1.5rem;
```
Comment (MuskanChhatrasal) [inline src/components/aside/aside.module.css:2]: Remove unnecessary spaces.

File: src/page/liked/liked.module.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  margin-right: 0.4rem;
+  margin-top: 0.2rem;
+}
+.iconWrapper > div:hover {
+  background-color: var(--COLOR-BACKGROUND-LIGHTER);
+  border-radius: 50%;
+}
+.toggleMenu {
+  background-color: var(--COLOR-BACKGROUND-LIGHTER);
+  display: flex;
```
Comment (MuskanChhatrasal) [inline src/page/liked/liked.module.css:58]: Make flex a helper class so that you don't have to write every time.

**Review by MuskanChhatrasal — APPROVED** [review]
Overall great work with the great UI. Just a few suggestions  - try to avoid inline styling, remove unnecessary spaces. Else everything looks good to merge.

**Review by partha8 — APPROVED** [review]
The code is clean, the way you broke The Like page and had liked card was good. Good readable code.

A suggestion, have cards like vertical/horizontal and name them that way. Here you have a "LikedCard" component, but thinking about reusability, having a general card will save time when you want to make pages like "watch later", "history". This way, you just need to make a few logical changes in that particular component.

File: src/components/aside/aside.module.css (hunk 1 above)
Comment (Developers-codz) [reply src/components/aside/aside.module.css:2]: okay

File: src/page/liked/liked.module.css (hunk 2 above)
Comment (Developers-codz) [reply src/page/liked/liked.module.css:58]: thank you for pointing that out  but I am using `module.css` only

**Comment by Developers-codz** [comment]
> The code is clean, the way you broke The Like page and had liked card was good. Good readable code.
> 
> A suggestion, have cards like vertical/horizontal and name them that way. Here you have a "LikedCard" component, but thinking about reusability, having a general card will save time when you want to make pages like "watch later", "history". This way, you just need to make a few logical changes in that particular component.

Very good suggestion will definitely try to generalizing the card, Thanks

**Pull request merged**